service/src/main/resources/application.yml

```diff
@@ -32,7 +32,7 @@ env:
 # contains the configuration of the BPM test application. We can use that application in our
 # integration testing to make sure the application code paths are working. However, we do not
 # want it to appear in production environments.
-spring.config.import: optional:file:service/build/resources/main/generated/local-properties.yaml
+spring.config.import: optional:file:build/resources/main/generated/local-properties.yaml
```

Comment:
Spring checks the classpath to find this file. My classpath included `.../terra-billing-profile-manager/service/`, but not `.../terra-billing-profile-manager/`, so having `service/` at the beginning of this file was preventing Spring from importing the config correctly since it was looking in `.../terra-billing-profile-manager/service/service/build/...`. This meant the config rendered by `render_configs.sh` couldn't be found and my local BPM wasn't able to successfully call Azure since the client ids and secrets were wrong.

Not sure if this was just a bug or if it was specific to my machine. If a reviewer would be so kind as to try this out, I'd appreciate it!
steps to repro:
1. render configs `./render_configs.sh local`
2. run BPM
3. call the `listManagedApps` API - http://localhost:8080/swagger-ui.html#/Azure/getManagedAppDeployments

Reply:
@aherbst-broad you inserted "service" here. Can you take a look?

https://github.com/DataBiosphere/terra-billing-profile-manager/pull/43/files#r976541391

Reply:
I think the issue is intellij not using the correct working directory by default when running the app. Maybe we can create a default Intellij configuration similar to WSM to avoid this. Anyway, I think this change is fine, I can run successfully using the new path.